CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.13.0 / 2019-10-04
+
+* [BUGFIX] UI: fix for [CVE-2019-10215](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-10215). #6098
+
 ## 2.13.0-rc.0 / 2019-09-25
```

Comment:
I don't think we track RC releases, do we?

Reply:
so it all should go under `2.13.0`

Reply:
We don't.

Reply:
yep , updated.